### ISSUE TITLE
square grid has gaps when using mask if unit size is very small

### ISSUE THREAD

**Issue by bribrah** [Turfjs/turf#1804]
TURF VERSION 5.1.6

I am trying to get a grid of very small squares over a small area of land. The issue is when I try and do
 ```
let squareGrid = turf.squareGrid(bbox, 0.01, {mask: polygon});
```
I get something that looks like this:
![bug](https://user-images.githubusercontent.com/43225721/70940094-76974200-1ffe-11ea-9083-21673e9de3f5.PNG)
whereas if i do it like this:
```
let squareGrid = turf.squareGrid(bbox, 0.0115, {mask: polygon});
```

I get this:
![nobug](https://user-images.githubusercontent.com/43225721/70940253-c970f980-1ffe-11ea-98b5-669bd7820132.PNG)

The smaller the units get the more gaps there are (.0115 is the smallest I can make it without getting gaps

**Comment by bribrah** [comment]
Another example:
```let squareGrid = turf.squareGrid(bbox, 0.005, {mask: polygon});```
![image](https://user-images.githubusercontent.com/43225721/71015087-e01f5b00-20a7-11ea-9856-07bea4f9ba81.png)

**Comment by rowanwins** [comment]
Hi @bribrah 

I think I've resolved this in the v7 alpha release - are you able to post a jsfiddle or equivalent for me to test on?

**Comment by bribrah** [comment]
Basically just ```let squareGrid = turf.squareGrid(bbox, 0.005, {mask: polygon});``` where polygon is any turf.polygon will result in this phenomena

**Comment by sprattek** [comment]
Jsfiddle example is available in #1215, [link]([http://jsfiddle.net/winsent/bh8mdefx/14/)

**Comment by mfedderly** [comment]
Closing per the above comment about this being resolved in 6.2.0.
